viewer/add-heading-anchors.js

```diff
@@ -143,6 +146,9 @@ var addHeadingAnchors = {
         window.location.hash = hash;
       }
     });
+  },
 
+  registerPostCopy: function(callback) {
+    $(this.target).find("a.headerLink").click(callback);
```

Comment:
sadly, i found this post copy stuff wasn't actually needed with the way we were doing things.  :x